consensus/src/counters.rs

```diff
@@ -526,14 +533,20 @@ pub static BLOCK_TRACING: Lazy<HistogramVec> = Lazy::new(|| {
     .unwrap()
 });
 
+const CONSENSUS_WAIT_DURATION_BUCKETS: [f64; 17] = [
+    0.02, 0.04, 0.06, 0.08, 0.10, 0.125, 0.15, 0.175, 0.2, 0.225, 0.25, 0.3, 0.4, 0.6, 0.8, 2.0,
```

Comment:
is the unit second here? if so this might be too large, most of time we see less than 10ms wait duration (this is mostly time drift between nodes)

Reply:
Yes. The unit is seconds. I added more buckets with lower time now. Balaji mentioned that we are currently facing problem with properly quantifying the anomalies, and we don't care about non-anomalies that much. I found some anomalies as much as 0.8 second when I looked into grafana for last 6 months. So, I added more points on the higher end.